versioned_docs/version-1.0.0/configuration.md

```diff
@@ -0,0 +1,152 @@
+---
+id: configuration
+slug: /configuration
+sidebar_position: 3
+---
+
+# Configuration
+
+If there is a need to modify the default configuration of MoonGuard, the
+configuration file can be published using:
```

Comment:
This can be written also as:

"If you need to modify the default configuration..."